url75694980d5_fabioalmeid_tp01ftc_tgz-pJ8-agentes_jade_Centralizador_ExemploParserGramaticaCentralizadorJ8/src/agentes/jade/Paciente/Paciente.java

```diff
@@ -0,0 +1,117 @@
+package agentes.jade.Paciente;
+
+public class Paciente {
+	private final static int MIN_TEMP = 35;
+	private final static int MAX_TEMP = 40;
+	
+	private final static int MIN_HEMO = 9;
+	private final static int MAX_HEMO = 23;
+	
+	private final static int MIN_BILI = 0;
+	private final static int MAX_BILI = 14;
+	
+	private static Integer temperatura = 39;
+	private static Boolean remedioTemp = false;
+	
+	private static Integer hemoglobina = 8;
+	private static Boolean remedioHemoglobina = false;
+	
+	private static Integer bilirrubina = 9;
+	private static Boolean remedioBilirrubuna = false;
+	
+	private static Boolean remedioPressao = false;
+
+	public static void setPressao(Integer diast, Integer sist) {
+		PressaoArterial.setSist(sist);
+		PressaoArterial.setDiast(diast);
+	}
+	
+	public static Integer getPressaoDiast() {
+		return PressaoArterial.getDiast();
+	}
+	
+	public static Integer getPressaoSist() {
+		return PressaoArterial.getSist();
+	}
+	
+	public static Integer getHemoglobina() {
+		return hemoglobina;
+	}
+	
+	public static Integer getHemoglobinaBoa() {
+		// 13,5 - 18 bom
+		// abaixo 12.5 anemia, acima de 18 doente
+		int Min = 14, Max = 18;
+		hemoglobina = Min + (int) (Math.random() * ((Max - Min) + 1));
+		return hemoglobina;
+	}
+
+	public static void setHemoglobina(Integer _hemoglobina) {
+		if ( (_hemoglobina >= MIN_HEMO) && (_hemoglobina <= MAX_HEMO)) 
+			hemoglobina = _hemoglobina;
+		else if (hemoglobina < MIN_HEMO)
+			hemoglobina = MIN_HEMO;
+		else if (hemoglobina > MAX_HEMO)
+			hemoglobina = MAX_HEMO;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 50:55, 63:68, 76:81.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.